dashboard/stack.yml

```diff
@@ -24,6 +24,7 @@ functions:
       # Comment out if not using public pretty-URL
       pretty_url: https://user.o6s.io/function
       query_pretty_url: 'true'
+      cookie_root_domain: '.system.o6s.io'
```

Comment:
This needs to be documented since we have the value in more than one place.

Reply:
Done. Can you check if it's enough?

Reply:
@martindekov WDYT?

Reply:
I approve the naming, if that is what you ask, it is pretty self explanatory

Reply:
If you are deploying it locally, informations from `auth/README.md` applies, and when you assign a domain to your localhost via `/etc/hosts` and you want to test auth locally, then you need to add that assigned domain here also.